wave_docs/index.mdx

```diff
@@ -22,26 +22,9 @@ Wave integrates with Seqera Platform credentials management enabling seamless ac
 Regulatory and security requirements sometimes dictate specific container images, but additional context is often needed.
 Wave enables any existing container to be extended without rebuilding it. Developers can add user-provided content such as custom scripts and logging agents, providing greater flexibility in the container’s configuration.
 
-Wave offers a flexible approach to container image management. It allows you to dynamically add custom layers to existing docker images, creating new images tailored to your specific needs.
-
-#### An example of Wave augmentation
```

Comment:
Did you mean to delete this section?

Reply:
@ewels, it came over as-is. @munishchouhan, did we want to remove this section from the Wave docs?

Reply:
Looks like that didn't happen here: https://github.com/seqeralabs/wave/pull/447

So this change occurred elsewhere.

Reply:
And we nuked the repository, so finding this is an interesting process of indirection, because the world started on Thu Mar 28 20:05:36 2024 +0100.

Reply:
i think its correct because the previous paragraph is stating the same as this

Reply:
@munishchouhan, ultimately what seems to have happened is part of this was added into the `docs` repo, but never made it back into the `wave` repo. So I'm going to add back in the deleted content, and then copy that back over to the `wave` repo so they're back in sync again.